docs/blog/2018-07-16-swag-store/index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Free Swag for Gatsby Contributors
+date: "2018-07-30"
+image: images/gatsby-swag.jpg
+author: Jason Lengstorf
+tags: ["collaboration","gatsby","community"]
+---
+
+Today, we’re _so excited_ to announce the launch of the [Gatsby Swag Store][store]! If you’ve been following along [on Twitter][twitter], you may have seen the news a little while back.
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">🎉The Gatsby Store is officially open! 🎉<br><br>We’ve got snuggly socks, soft t-shirts, and enough stickers to wallpaper your house. 🧦👕🏡<br><br>Get ’em at <a href="https://t.co/e7xq9qTaP7">https://t.co/e7xq9qTaP7</a><br><br>P.S. Do you <a href="https://twitter.com/hashtag/buildwithgatsby?src=hash&amp;ref_src=twsrc%5Etfw">#buildwithgatsby</a>? All contributors get FREE SWAG. Sign in to claim your discount code!</p>&mdash; Marisa Morby (@marisamorby) <a href="https://twitter.com/marisamorby/status/1019256499799912449?ref_src=twsrc%5Etfw">July 17, 2018</a></blockquote>
+
+We’ve spent the last couple weeks working the kinks out, and we couldn’t be happier with the response we’ve seen.
+
+<figure>
+  <img alt="Gatsby swag posts on Twitter." src="./images/gatsby-swag-twitter.jpg" />
+  <figcaption>
+    Tweets from <a href="https://twitter.com/DeaNHtiD99/status/1023204484183416832">Dean</a>, <a href="https://twitter.com/ARebelBelle/status/1020044426712735744">Elle</a>, <a href="https://twitter.com/chaseadamsio/status/1021896138503245824">Chase</a>, and <a href="https://twitter.com/kato_katherine/status/1021967765400211456">Katherine</a> showing off their new Gatsby swag.
+  </figcaption>
+</figure>
+
+## Free swag for all contributors 💪💜
+
+We’re most excited about the launch of the store because it means contributors are now able to get stickers, socks, and/or t-shirts for free — a little token of our appreciation for putting the time in to make the open source community even better. You’re all amazing, and we 💜 you for it.
```

Comment:
I'd take out the word *most* unless you plan on listing the other reasons we're excited